frontend/src/app/app.module.ts

```diff
@@ -11,35 +11,35 @@ import { CommonModule } from '@angular/common';
 import { HTTP_INTERCEPTORS, HttpClientModule } from '@angular/common/http';
 import { NgModule } from '@angular/core';
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
```

Comment:
Issue: 
- View diagrams should be larger and bold
- Margin left & right of the Download button is not the same.
- Gap between mat cards is missing.

Screenshot:
![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/cfd23925-6dce-4db7-ad70-a633b90978da)


PS: If you need help to set up the diagram cache, let me know.